client/features/auth-pages/SignUpPage.tsx

```diff
@@ -94,16 +82,15 @@ const SignUpPage = () => {
   };
 
   const onSubmit: SubmitHandler<FormData> = async (data) => {
-    const { email, password, name } = data;
```

Comment:
Consider using the useCreateUserProfileMutation fx here to create a new userProfile for  when they sign up by email and password. Currently no username is displayed when we sign up by email and password. Keep in mind a userProfile should only be created after a user has been successfully authenticated via firebase.